packages/base-styles/_variables.scss

```diff
@@ -104,3 +104,18 @@ $block-selected-to-content: $block-edge-to-content - $block-selected-margin - $b
 $radius-round-rectangle: 4px;
 $radius-round: 50%;
 $radius-block-ui: 2px;
+
+// Ideally this is not repeated on all built styles
+// But until we drop IE11 support, it's need to generate fallback colors.
+:root {
+	--wp-admin-theme-color: theme(primary);
```

Comment:
These `theme()` calls are generating incorrect selectors for the custom properties— I'm seeing `body.admin-color-ocean :root`, when it should be just `body.admin-color-ocean`, etc

Reply:
Yes, I saw that, I'm leaning towards removing the post css plugin but we can only do that if we drop IE11 support. Otherwise, I'll have to fix the Postcss plugin.